helm/chart/README.md

```diff
@@ -13,7 +13,7 @@ checks
 | `scheduler.scheduler.containerSecurityContext`                                   | Sets security context                                          |                           |
 | `scheduler.scheduler.env.otelCollectorUrl`                                       | sets url for open telemetry collector                          | `otel-collector:4317`     |
 | `scheduler.scheduler.image.repository`                                           | set image repository for scheduler                             | `ghcr.io/keptn/scheduler` |
-| `scheduler.scheduler.image.tag`                                                  | set image tag for scheduler <!---x-release-please-version-->   | `v0.8.1`                  |
+| `scheduler.scheduler.image.tag`                                                  | set image tag for scheduler                                    | `v0.8.1`                  |
```

Comment:
release please comment missing?

Reply:
in multiple place in the file 😢

Reply:
We don't need the release please comments anymore. Since we have a monorepo now, renovate will take of updating the sub-projects for us. No need to do it on release anymore.